packages/@ember/utils/lib/type-of.ts

```diff
@@ -98,7 +98,7 @@ const { toString } = Object.prototype;
   @public
   @static
 */
-export function typeOf(item: unknown): TypeName {
+function typeOf(item: unknown): TypeName {
```

Comment:
Question: any reason to prefer the *separate* `export default` vs. just doing this?

```suggestion
export default function typeOf(item: unknown): TypeName {
```